code/game/objects/items/weapons/defib.dm

```diff
@@ -450,15 +455,20 @@
 					for(var/obj/item/organ/external/O in H.bodyparts)
 						total_brute	+= O.brute_dam
 						total_burn	+= O.burn_dam
-					if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !ghost && tplus < tlimit && !HAS_TRAIT(H, TRAIT_HUSK)  && !HAS_TRAIT(H, TRAIT_BADDNA) && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)))
+					if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !ghost && tplus < tlimit && !HAS_TRAIT(H, TRAIT_HUSK) && !HAS_TRAIT(H, TRAIT_BADDNA) && H.blood_volume > BLOOD_VOLUME_SURVIVE && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)))
 						tobehealed = min(health + threshold, 0) // It's HILARIOUS without this min statement, let me tell you
 						tobehealed -= 5 //They get 5 of each type of damage healed so excessive combined damage will not immediately kill them after they get revived
 						H.adjustOxyLoss(tobehealed)
 						H.adjustToxLoss(tobehealed)
 						H.adjustFireLoss(tobehealed)
 						H.adjustBruteLoss(tobehealed)
-						user.visible_message("<span class='boldnotice'>[defib] pings: Resuscitation successful.</span>")
-						playsound(get_turf(src), 'sound/machines/defib_success.ogg', 50, 0)
+						if(H.get_int_organ(/obj/item/organ/internal/brain) && H.getBrainLoss() >= 100)
+							// If you want to treat this with mannitol, it'll have to metabolize while the patient is alive, so it's alright to bring them back up for a minute
+							playsound(get_turf(src), 'sound/machines/defib_saftyoff.ogg', 50, 0)
+							user.visible_message("<span class='boldnotice'>[defib] chimes: Minimal brain activity detected, brain treatment recommended for full resuscitation.</span>")
+						else
+							user.visible_message("<span class='boldnotice'>[defib] pings: Resuscitation successful.</span>")
```

Comment:
You may want to move this line out of the `else` section and only have the `playsound` in there - the resuscitation _is_ successful, after all, the doctor should probably know that they successfully revived the person (with the added warning, of course).

Not sure how spammy it would get, though, but it may worth a test.

Reply:
Done, I agree it makes sense to give the feedback regardless.